example-app/src/util/paymentMethodTypes.ts

```diff
@@ -0,0 +1,82 @@
+import type { IPaymentMethodType } from '../types';
+
+export const DEFAULT_ENABLED_PAYMENT_METHOD_TYPES = ['card_present'];
+
+export const PAYMENT_METHOD_TYPES = [
```

Comment:
Nit: Given these these variable is only used in 1 place and the final goal is to generate `IPaymentMethodType`, should we hide these and expose the `IPaymentMethodType[]` directly?
If not, maybe a rename on these field can avoid misunderstanding it's not a type but a string.

Reply:
changed to not use type